index.html

```diff
@@ -170,7 +170,14 @@ <h2 class="card_name">{card_name}</h2>
     </div>
 
     <div id="top_of_deck" class="card">
-        Click to Start
+        Click to Start<br>
+        Order of Opperations<br>
+    	If engaged and standing<br>
+	    If not engaged and standing<br>
```

Comment:
This isn't the 'right' way to do it, but if you want your "tab" or "spaces" to render on the browser (I'm assuming it's just collapsing them and it doesn't look like you have an indentation) you can substitute:

&nbsb;

instead of a tab or a space. 
https://www.educative.io/answers/what-is-nbsp-in-html

Reply:
^ so for four spaces you can do:

&nbsb;&nbsb;&nbsb;&nbsb;